build/linux/installer/conf/fluent-bit.conf

```diff
@@ -47,5 +47,18 @@
     Match oms.container.log.la.*
     Exclude stream ${AZMON_LOG_EXCLUSION_REGEX_PATTERN}
 
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
+#${KubernetesFilterEnabled}    Merge_Log           Off
+#${KubernetesFilterEnabled}    Buffer_Size         0
+#${KubernetesFilterEnabled}    Use_Kubelet         true
+#${KubernetesFilterEnabled}    Kubelet_Port        10250
+#${KubernetesFilterEnabled}    Kubelet_Host        ${NODE_IP}
+#${KubernetesFilterEnabled}    tls.verify          Off
+#${KubernetesFilterEnabled}#${AnnotationBasedLogFilteringEnabled}    K8S-Logging.Exclude On
+#${KubernetesFilterEnabled}    Kube_Meta_Cache_TTL ${AZMON_KUBERNETES_METADATA_CACHE_TTL_SECONDS}
+#${KubernetesFilterEnabled}    Kube_Tag_Prefix     oms.container.log.la.var.log.containers.
```

Comment:
same here .. Doesnt this needs to contain "C.var.log.container"  like in other places?

Reply:
for win only